src/persist/src/indexed/encoding.rs

```diff
@@ -866,6 +867,96 @@ impl From<&Description<u64>> for ProtoU64Description {
     }
 }
 
+/// Encodes the inline metadata for an unsealed batch into a base64 string.
+pub fn encode_unsealed_inline_meta(batch: &BlobUnsealedBatch, format: ProtoBatchFormat) -> String {
+    let inline = ProtoBatchInline {
+        format: format.into(),
+        batch_type: Some(proto_batch_inline::Batch_type::unsealed(
+            ProtoUnsealedBatchMeta {
+                seqno_lower: batch.desc.start.0,
+                seqno_upper: batch.desc.end.0,
+
+                // Save a few bytes by not including these.
+                key: Default::default(),
```

Comment:
These are empty/default because batches don't store their meta-data themselves, right? (we store it in the Arrangement meta proto). Could be a sign that we should use a separate proto to encode this inline information, but I probably don't have enough context.

Reply:
yeah, this is something I want to hash out with your and ruchir. there's a bit of context in https://github.com/MaterializeInc/materialize/pull/9895

Reply:
Did what we agreed on in the sync today